pkgs/development/libraries/libav/default.nix

```diff
@@ -118,7 +118,7 @@ let
         "--cross-prefix=${stdenv.cc.targetPrefix}"
         "--enable-cross-compile"
         "--target_os=linux"
-        "--arch=${hostPlatform.arch}"
+        "--arch=${hostPlatform.parsed.cpu.name}"
```

Comment:
Why do these attribute names contain "parsed"? Doesn't make much sense to me.

Reply:
In trying to reconcile `crossSystem`, `system`, `platform`, and @nbp's previously-dead parsing code, yes I had to find some awkward unused namespace at first to avoid breaking things. (`hostPlatform.arch` is inconsistent, but closer to `hostPlatform.platform.gcc.arch`.) Once I do changes like this, I get closer to being able to drop the `parsed` and move everything up a level.